content.en/polkadot/dht/2024-23-report.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Week 2024-23
+plotly: true
+slug: 2024-23
+weight: 1045636
+---
+
+# Polkadot Report Cal. Week 23 - 2024
+
+## General Information
+
+The following results show measurement data that were collected in calendar week 23 in 2024 from `2024-06-03` to `2024-06-10`.
+
+- Number of crawls `336`
+- Number of visits `7,693,028`
+  > Visiting a peer means dialing or connecting to it. Every time the crawler or monitoring process tries to dial or connect to a peer we consider this as _visiting_ it. Regardless of errors that may occur.
+- Number of unique peer IDs visited `11,027`
+- Number of unique peer IDs discovered in the DHT `10,655`
+- Number of unique IP addresses found `16,947`
+
+Timestamps are in UTC if not mentioned otherwise.
+
+## Churn Analysis
+
+This visualizes the uptime of peers over a specific period of time. The plot may display trends in churn rate, distribution of uptime periods, or patterns in peer activity. It helps analyze the stability and reliability of the network by identifying fluctuations in peer participation and examining potential factors influencing churn.
+
+> 💡 Due to the diversity of Polkadot agents and agent versions, we've only included agents that have more than 50 peers.
+
+{{< plotly json="../../../plots/2024/06/03/polkadot-plot-churn.json" height="600px" >}}
```

Comment:
I have not looked at the code use to generate the churn plot yet, but I always found this plot weird. It seems to underestimate the uptime of peers.

In this plot, it seems weird that after 2 hours, 74% of all Polkadot nodes seem to have churned away. I would expect the network to be much more stable than what this plot is showing.